dev-util/stripe-mock/stripe-mock-0.34.1.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
@gyakovlev, could you review this, please?

Reply:
The `golang-build` eclass installs a bunch of `.a` files and all the sources of vendored libraries. Is it supposed to be done this way? I would expect that such things are hidden behind `USE="static-libs"` and `USE="src"`, respectively.

Reply:
Yes, go is static-only crap. I have no clue about sources but I suppose leave it as-is; @gyakovlev ?

Reply:
Other ebuilds such as [go-protobuf-1.2.0](https://github.com/gentoo/gentoo/blob/master/dev-go/go-protobuf/go-protobuf-1.2.0.ebuild) seem to remove the bundled sources in `src_install`.

I personally don't like this solution, so I am not going to touch `src_install` for now.

Reply:
BTW I find it kinda weird that it installs all that stuff and not the actual executable.

Reply:
Hi!

it depends not how package is used but generally you don't need it and it's perfectly fine to just `dobin` the binary and accompanying required files (e.g. configs, docs) and be done with it.

`golang-src_install` and the sources/libraries it installs are needed if you your package is a dependency of another go package you are packaging and you want to use system-wide installed go sources instead of temporary bundling stuff during build time and discarding right after.

Same applies to libraries, they are used if you are compiling something against given package, but generally not used at runtime by the package itself.

TLDR: if you don't plan to build any go package that requires `stripe-mock` sources installed system-wide then ditch the sources and probably libraries as well.

Reply:
Thanks for the clarification!
I can't foresee the the future, but I find it unlikely that someone will build upon stripe-mock. Therefore, I decided to remove the sources and libraries from the GOPATH directory.